feature/chat/src/main/kotlin/uk/gov/govuk/chat/ui/ChatScreen.kt

```diff
@@ -148,6 +151,7 @@ private fun ChatContent(
                     modifier = Modifier
                         .background(GovUkTheme.colourScheme.surfaces.chatBackground)
                         .padding(all = GovUkTheme.spacing.medium)
+                        .semantics { isTraversalGroup = true }
```

Comment:
From everything I've seen this implementation should be correct, but it does not work as desired. I've tried multiple approaches to rectify: new AVD, clear caches, different AVD, different API level, wipe AVD data, removing the AnimatedVisibility, removing everything but the action button and the text input ... Nothing seems to solve it. I've created a minimal test - that works, so It must be something in the code - I just can't seem to find it.